### PR TITLE
Add guest_uuid_change function for hyperv mode

### PR DESCRIPTION
**Pull request by warku123**

### CODE REVIEW

File: hypervisor/virt/hyperv/hypervcli.py [hunk 1]
```diff
@@ -261,3 +261,45 @@ def guest_resume(self, guest_name):
         else:
             logger.error("Failed to resume hyperv guest")
             return False
+
+            
+    def hypervisor_guid(self):
```
Comment (hkx303) [inline hypervisor/virt/hyperv/hypervcli.py]: we already have this function, see https://github.com/VirtwhoQE/hypervisor-builder/blob/master/hypervisor/virt/hyperv/hypervcli.py#L96

File: hypervisor/virt/hyperv/hypervcli.py [hunk 2]
```diff
@@ -261,3 +261,45 @@ def guest_resume(self, guest_name):
...
+            logger.info("Succeeded to get hypervisor guid")
+            return output.strip()
+        else:
+            logger.error("Failed to get hypervisor guid")
+            return ""
+    
+    def hypervisor_change_guid(self, guid, guest_name):
+        """
+        Change guid for hypervisor
+        :param guid: the guid for hypervisor
```
Comment (hkx303) [inline hypervisor/virt/hyperv/hypervcli.py]: the guid for guest

File: hypervisor/virt/hyperv/hypervcli.py [hunk 3]
```diff
@@ -261,3 +261,45 @@ def guest_resume(self, guest_name):
...
+        cmd = r"PowerShell (gwmi -Namespace Root\Virtualization\V2 -ClassName Msvm_VirtualSystemSettingData).BiosGUID"
+        ret, output = self.ssh.runcmd(cmd)
+        if not ret:
+            logger.info("Succeeded to get hypervisor guid")
+            return output.strip()
+        else:
+            logger.error("Failed to get hypervisor guid")
+            return ""
+    
+    def hypervisor_change_guid(self, guid, guest_name):
```
Comment (hkx303) [inline hypervisor/virt/hyperv/hypervcli.py]: rename to guest_uuid_change()

**Review by hkx303 — COMMENTED** [review]
Hello, pinge, reviewed, please update, thanks :)

File: hypervisor/virt/hyperv/hypervcli.py (hunk 1 above)
Comment (warku123) [reply hypervisor/virt/hyperv/hypervcli.py]: OK. I see

File: hypervisor/virt/hyperv/hypervcli.py [hunk 4]
```diff
@@ -261,3 +261,30 @@ def guest_resume(self, guest_name):
...
+        """
+        create_function_cmd = r"PowerShell (Invoke-WebRequest http://10.73.131.85/ci/hyperv/New-VMBIOSGUID.ps1 -OutFile ./New-VMBIOSGUID.ps1)"
+        ret, _ = self.ssh.runcmd(create_function_cmd)
+        if ret:
+            logger.error("Failed to create function")
+            return False
+        
+        import_module_cmd = r"Import-Module ./New-VMBIOSGUID.ps1 -Force"
+        set_ignore_verfiy_cmd = r"$ConfirmPreference = 'None'"
+        change_guid_cmd = 'PowerShell -Command "{}; {}; New-VMBIOSGUID -VM {} -NewID {}"'.format(set_ignore_verfiy_cmd,
```
Comment (hkx303) [inline hypervisor/virt/hyperv/hypervcli.py]: Hello Pinge, please use `f-string` instead of `format`, thanks

**Review by hkx303 — APPROVED** [review]
Ack, with a small comment

**Review by hkx303 — APPROVED** [review]
Thanks for the PR and quick update!

**Pull request merged**